6-refactoring/exercises/dom-and-events/mouse-coordinates/src/handler.js

```diff
@@ -1,3 +1,17 @@
-import { formatCoordinates } from './utils.js';
+import { formatCoordinates } from './util.js';
 
-export const showMouseCoordinates = () => {};
+export const showMouseCoordinates = (event) => {
+        debugger;
+        // read & process user input
+        const xValue = event.pageX;
+        const yValue = event.pageY;
+        // execute core logic
+        const formattedCoordinates = 'X: ' + xValue + '\nY: ' + yValue;
```

Comment:
Use the function `formatCoordinates` of the `util.js`.

Ex:

```suggestion
        const formattedCoordinates = formatCoordinates(xValue, yValue);
```